### DISCUSSION TITLE
Problem with connecting to mysql database connect through shh by port forwarding.

### DISCUSSION THREAD

**Discussion by SubhashiniMadhavanExporo** [prisma/prisma#13195] [Q&A]
### Bug description

prod database is inside the bastion server and it has a VPC. It can only be connected via ssh port forwarding technique. So, I forward the database in the bastion server host to local host via using `ssh -i [ssh_pubkey] -L portnumber:bastionhost:port number -- user@localhost`. and then try to connect to the localhost it has trouble connecting

### Prisma information

An then give the `DATABASE_URL = "mysql://root:pwd@localhost:3306/database"`

run the following command `prisma:migrate:create`:

and then I get this error: 
```
Error: P1000: Authentication failed against database server at `localhost`, the provided database credentials for `root` are not valid
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```
6.14.11

```

**Comment by janpio** [comment]
Can you connect with another MySQL client with that connection string?
What is your Prisma version? 6.14.11 does not look right - we are at 3.13.0 right now.

**Reply by SubhashiniMadhavanExporo** [reply]
I can connect to mysql workbench with same database credentials and host but I could not connect with database _Url provided above. 

Sorry for it Prisma version is 3.2.1

**Reply by pantharshit00** [reply]
Hello @SubhashiniMadhavanExporo 

Is this still relevant? I wasn't able to reproduce this with 4.7.0

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
